README.md

```diff
@@ -67,6 +69,7 @@ __*Note - Each Operator version is compatible with all corresponding OCI artifac
 ## Open Enterprise SDK for Python <a name="python"></a>
 |  Product Version | OCI artifact | License | Find out more | 
 |-|-|-|-| 
+| 3.10.0 | 3.10.0.2 | [Read here](https://www14.software.ibm.com/cgi-bin/weblap/lap.pl?li_formnum=L-JYIP-C7JT5W) | [Release notes](https://www.ibm.com/docs/en/SSV97FN_2022.1.1/python/release_note.html) |
 | 3.10.0 | 3.10.0.1 | [Read here](https://www14.software.ibm.com/cgi-bin/weblap/lap.pl?li_formnum=L-JYIP-C7JT5W) | [Release notes](https://www.ibm.com/docs/en/SSV97FN_2022.1.1/python/release_note.html) |
 | 3.10.0 | 3.10.0.0 | [Read here](https://www14.software.ibm.com/cgi-bin/weblap/lap.pl?li_formnum=L-JYIP-C7JT5W) | [Release notes](https://www.ibm.com/docs/en/SSV97FN_2022.1.1/python/release_note.html) |
```

Comment:
All these Release Note links will always point to the latest doc changes, which may not have the specific release notes for the product version in the table row. Is this the intention? If so - should you just consider a top-level release notes link outside of the table? Since, I can't see older release notes anyway?

Reply:
Good question, I will check with Nate. There's some consideration of adding the _changelog/what's new_ for older releases [here](https://github.ibm.com/IBM-Z-and-Cloud-Modernization-Stack/tracker/issues/617#issuecomment-47196432).  We could also link to the what's new page in the product's documentation for older releases.

Reply:
I have gone ahead with the top-level link